### ISSUE TITLE
Master picture filename converted to underscore resulting in ENOENT

### ISSUE THREAD

**Issue by dsamarin** [RealFaviconGenerator/rfg-api#25]
I was getting the following message:

```sh
Error: ENOENT: no such file or directory, open 'assets/raw/_logo__icon.svg'
```

when I was using the API with the filename: 

```js
const icon = "assets/raw/Logo_Icon.svg";
// ...
        windows: {
          masterPicture: icon,
// ...
```

So adding `'master_picture',` to the `keysToIgnore` array from `camelCaseToUnderscoreRequest` solved this annoying problem.

**Comment by Jelle-S** [comment]
Same issue here. Is this library still maintained?

**Comment by phbernard** [comment]
It's fixed in 0.5.3. Sorry for the long wait.
